include/revng/Model/Types.h

```diff
@@ -0,0 +1,335 @@
+#pragma once
```

Comment:
What about `TypeSystem.h`? Otherwise `Type.h` is also OK.

Reply:
I'll go for `Type.h`

Reply:
Done